src/DotNet.Status.Web/Controllers/AzurePipelinesController.cs

```diff
@@ -26,33 +30,48 @@ public class AzurePipelinesController : ControllerBase
     {
         private readonly IGitHubApplicationClientFactory _gitHubApplicationClientFactory;
         private readonly IAzureDevOpsClientFactory _azureDevOpsClientFactory;
+        private readonly IMaestroApiClientFactory _maestroClientFactory;
         private readonly IOptions<BuildMonitorOptions> _options;
+        private readonly IOptions<MaestroOptions> _maestroOptions;
         private readonly ILogger<AzurePipelinesController> _logger;
         private readonly Lazy<IAzureDevOpsClient> _clientLazy;
+        private readonly Lazy<IMaestroApi> _maestroApiLazy;
         private readonly Lazy<Task<Dictionary<string, string>>> _projectMapping;
 
         public AzurePipelinesController(
             IGitHubApplicationClientFactory gitHubApplicationClientFactory,
             IAzureDevOpsClientFactory azureDevOpsClientFactory,
+            IMaestroApiClientFactory maestroApiClientFactory,
             IOptions<BuildMonitorOptions> options,
+            IOptions<MaestroOptions> maestroOptions,
```

Comment:
This is going to be another secret rotation problem. :-)

Reply:
Will fix it up. The code was written before you started making changes. I will look at what you've done and adjust